src/main/kotlin/dev/arbjerg/ukulele/audio/Player.kt

```diff
@@ -116,6 +123,12 @@ class Player(val beans: Beans, guildProperties: GuildProperties) : AudioEventAda
         player.playingTrack.position = position
     }
 
+    override fun onTrackStart(player: AudioPlayer, track: AudioTrack) {
+        if (beans.botProps.announceTrack) {
+            lastChannel?.sendMessage(beans.nowPlayingCommand.buildEmbed(track))!!.queue()
```

Comment:
The `!!` operator will trigger an error because the receiver is null if `lastChannel` is null.

```suggestion
            lastChannel?.sendMessage(beans.nowPlayingCommand.buildEmbed(track))?.queue()
```